tsconfig.json

```diff
@@ -14,5 +14,9 @@
   },
   "include": [
     "src/**/*"
+  ],
+  "exclude": [
+    "tests/**/*",
```

Comment:
Is this needed if only `src` is included?
I guess someone would have to explicitly import from the tests folder.

Reply:
It's actually not required :partying_face:  but removing *include* would cause it to attempt to build tests strangely.